app/src/main/java/com/pajato/android/gamechat/chat/adapter/ChatListAdapter.java

```diff
@@ -243,6 +251,9 @@ private void updateChatHolder(ChatListViewHolder holder, final ChatListItem item
             text = (TextView) itemView.findViewById(R.id.chatText);
             icon = (ImageView) itemView.findViewById(R.id.chatIcon);
             checkBox = (CheckBox) itemView.findViewById(R.id.selectorCheck);
+            if (checkBox != null) {
+                checkBox.setOnClickListener(new ChatListCheckBoxClickListener());
```

Comment:
You might want to cache a click handler rather than creating a new one every for every list item.  The garbage collector will love you for it almost as much as me! :-)